src/utils/convertToCsv.ts

```diff
@@ -3,7 +3,7 @@
  * @param data - the array of JSON objects to convert.
  */
 export const convertToCSV = (
-  data: any,
+  data: Array<any>,
```

Comment:
This is a completely visual thing, but in the rest of the project I think we use `Type[]` or in this case `any[]`. Or if I am wrong if it is the other way around, we should put whichever is more consistent.

Reply:
good point